tests/commands/realm.py

```diff
@@ -63,5 +63,16 @@ def runTest(self):
 realm join blah1
 realm join blah2""")
 
+class F42_TestCase(F19_TestCase):
+    def runTest(self):
+        F19_TestCase.runTest(self)
+
+        # Support the --computer-name option
+        self.assert_parse("""realm join --computer-name=blee blah2""")
+
+        # Support the --computer-name option alongside another supported argument 
+        self.assert_parse("""realm join --computer-name=blee --computer-ou=OU=blah blah2""")
+
```

Comment:
Could you add a test for the error case if nothing is passed to it. eg. `realm join --computer-name`

Reply:
Yup - I've added added this, but it failed to produce an error with an empty argument so I added in a [check](https://github.com/dsnpl/pykickstart/blob/15b04f5a1d4d9d8ca7eaf24807d4a36d184410a4/pykickstart/commands/realm.py#L127) to ensure that a non empty argument was present for `--computer-name`. The default behaviour of getopt is not to error in this case.

Should I add this check for all options that require an argument? If so should I extend this to F19 as well?

Thanks.

Reply:
Oh, ha. Surprised nobody noticed that before. I'd check all options but just add it to F42, we try not to change behavior in old releases (especially EOL ones) unless there is a really good reason.